### DISCUSSION TITLE
[working directory] How to start WezTerm in a particular directory from the cli?

### DISCUSSION THREAD

**Discussion by 2brownc** [wezterm/wezterm#2797] [Q&A] [answered]
For example to start XFCE Terminal in '/var':

```
$ xfce4-terminal --working-directory=/var
```

But how do I start WezTerm in a particular directory. I tried some commands but it isn't working out:

```
$ wezterm set-working-directory /var
$ wezterm set-working-directory=/var
```

**Comment by wez** [comment, marked as answer]
`wezterm start --cwd /some/path`

See also: https://wezfurlong.org/wezterm/config/launch.html#specifying-the-current-working-directory

You can also run `wezterm --help` and `wezterm start --help` to see more information about the CLI.

**Reply by 2brownc** [reply]
This worked. Thank you.

**Reply by raffaelemancuso** [reply]
On Windows this open a Windows terminal together with Wezterm. Use `wezterm-gui start --cwd /some/path`
